www/front_src/src/Authentication/Openid/useValidationSchema.ts

```diff
@@ -0,0 +1,49 @@
+import { useTranslation } from 'react-i18next';
+import * as Yup from 'yup';
+
+import { OpenidConfiguration } from './models';
+import {
+  labelRequired,
+  labelInvalidIPAddressOrDomainName,
+  labelInvalidURL,
+} from './translatedLabels';
+
+const IPAddressAndDomainRegexp =
+  /^\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3}(\/\d{1,3})?$|^(\S)+\.{1,}\D+[^.]$/;
+
+const urlRegexp = /https?:\/\/(\S+)/;
```

Comment:
https, the "s" is mandatory on this feature ?

Reply:
It is not mandatory